core/src/main/java/com/radixdlt/utils/properties/RuntimeProperties.java

```diff
@@ -64,86 +64,273 @@
 
 package com.radixdlt.utils.properties;
 
-import java.util.Arrays;
-import org.apache.commons.cli.CommandLine;
-import org.apache.commons.cli.CommandLineParser;
+import static com.radixdlt.utils.RadixConstants.STANDARD_CHARSET;
+
+import com.radixdlt.utils.BooleanUtils;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.function.Function;
 import org.apache.commons.cli.DefaultParser;
-import org.apache.commons.cli.Option;
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.ParseException;
-import org.json.JSONObject;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 /**
- * Runtime properties are temporary property sets that are discarded upon client termination.
- *
- * <p>They combine a set of persisted properties, with a set of command line properties.
+ * Persisted properties are property sets that are committed to storage and may be reloaded on the
+ * next application execution.
  */
-public final class RuntimeProperties extends PersistedProperties {
-  private final CommandLine commandLine;
+public class RuntimeProperties {
```

Comment:
The diff is a bit misleading :) What happened here is:
- I moved `fromCommandLineArgs` from RuntimeProperties (consturctor) to PersistedProperties
- I removed the original RuntimeProperties and renamed PersistedProperties -> RuntimeProperties